hash

```diff
@@ -55,8 +56,12 @@ sparkSubmit \
   --master "${MASTER:=local[*]}" \
   --name "${app_name}" \
   --jars "${deps_jar}" \
+  --conf "spark.driver.memory=4g" \
```

Comment:
does it mean we require 4GB of memory to be given to docker?
If yes we need at least to document it. For "docker for mac" the default value is 1GB if I remember right.

Reply:
No, it does not. I just max capacity that the Driver JVM process possibly can take.